src/Core/Services/Implementations/UserService.cs

```diff
@@ -1087,13 +1087,24 @@ public async Task<bool> TwoFactorProviderIsEnabledAsync(TwoFactorProviderType pr
             return await CanAccessPremium(user);
         }
 
+        //TODO refactor this to use the below method and enum
```

Comment:
Where all is this referenced? Is it not possible to refactor and eliminate redundant code now vs. later?

Reply:
As far as I know just in enterprise, but given that references are using magic strings and any errors would only be found at runtime I wasn't sure if it was worth the risk of getting into it right now.

Reply:
We'll refactor soon after this PR gets in, don't worry about it now.